MH-FiorellaTheme5/source/javascripts/site.js

```diff
@@ -1 +1,8 @@
 // This is where it all goes :)
+function openNav() {
+    document.getElementById("mySidenav").style.width = "95%";
```

Comment:
Let's avoid modifying styles directly in the element using JS, instead of that create a class containing the `width:  95%` and set that class to the element by using something like:
```
const sideNav = document.getElementById('mySidenav');
sideNav.classList.add('your class with the style above');
```
And inside the method that closes the nav just remove the class you just added 🤓